From f587c57f7d64c1fd29653e48cf854fb20f2a29a8 Mon Sep 17 00:00:00 2001
From: BtbN <btbn@btbn.de>
Date: Sat, 17 May 2025 01:14:38 +0200
Subject: [PATCH] Fix build with gcc-15

Fixes:
src/zimg/graph/filtergraph.cpp:28:72: error: 'uintptr_t' does not name a type
---
 src/zimg/graph/filtergraph.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/zimg/graph/filtergraph.cpp b/src/zimg/graph/filtergraph.cpp
index f02803fe..351ea063 100644
--- a/src/zimg/graph/filtergraph.cpp
+++ b/src/zimg/graph/filtergraph.cpp
@@ -1,5 +1,6 @@
 #include <algorithm>
 #include <climits>
+#include <cstdint>
 #include <tuple>
 #include "common/align.h"
 #include "common/except.h"
